views/js/pciCreator/ims/audioRecordingInteraction/runtime/audioRecordingInteraction.js

```diff
@@ -134,7 +134,8 @@ define([
                     response = { file: this.getRecording() };
                 }
                 return {
-                    base: response
+                    base: response,
+                    recordsAttempts: this._recordsAttempts
```

Comment:
I think that adding the property to the response object here is not right - it's no longer a correctly-formatted QTI-JSON response (`{ base: { file: <string> } }`).

Try keeping the return value of `getResponse()` unchanged, but change the return value of `getState()` to include your new property. This new feature should be writing to and reading from the state, not the response.